<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExportSubProject</class>
 <widget class="QWidget" name="DialogExportSubProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>212</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>344</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sub Project Export</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_export">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QCheckBox" name="openExplorerFolderCheckBox">
     <property name="text">
      <string>Open folder after export</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_folder">
     <property name="placeholderText">
      <string>Enter output folder</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_folder">
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="toolButton_folder">
     <property name="text">
      <string>Select Out Folder</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/open_project.png</normaloff>:/icons/100x100/open_project.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="subProjectExportSettingsGroupBox">
     <property name="title">
      <string>Export settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QComboBox" name="objectsFilterComboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="subProjectInfoButton">
        <property name="text">
         <string>Sub Project Info</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Objects filter :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>objectsFilterComboBox</tabstop>
  <tabstop>subProjectInfoButton</tabstop>
  <tabstop>lineEdit_folder</tabstop>
  <tabstop>toolButton_folder</tabstop>
  <tabstop>openExplorerFolderCheckBox</tabstop>
  <tabstop>pushButton_export</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
